src/Native/System.Net.Http.Native/pal_easy.h

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#include "pal_types.h"
+
+#include <curl/curl.h>
+
+enum
+{
+    CurlOptionLongBase = 0,
+    CurlOptionObjectPointBase = 10000,
+    CurlOptionFunctionPointBase = 20000,
+};
+
+enum PAL_CURLoption : int32_t
+{
+    PAL_CURLOPT_INFILESIZE = CurlOptionLongBase + 14,
```

Comment:
Just to add to Jeremy's comments, libcurl is also a bit of a special case, in that it does publically document the values of most of its contants.  That's unusual when you look at other libraries we're consuming, where not only are the values not documented, the values change from implementation to implementation.